forward_engineering/helpers/databaseHelper.js

```diff
@@ -65,22 +66,32 @@ const getUseCatalogStatement = (databaseData) => {
  * @param {string} dbVersion
  * */
 const getDatabaseStatement = (containerData, isUnityCatalogSupports, dbVersion) => {
+    const UNITY_TAGS_RUNTIME_REQUIRED = '13';
 	const tab = getTab(0, containerData);
 	const name = replaceSpaceWithUnderscore(prepareName(getName(tab)));
 	if (!name) {
 		return '';
 	}
 
-	return getCreateStatement({
-		name: name,
-		comment: tab.description,
-		location: tab.location,
-		managedLocation: tab.managedLocation,
-		dbProperties: tab.dbProperties,
-		isActivated: tab.isActivated,
-		isUnityCatalogSupports,
-		dbVersion
-	});
+    let unityCatalogTags = '';
+    let unitySchemaTags = '';
+    const createStatement = getCreateStatement({
+        name: name,
+        comment: tab.description,
+        location: tab.location,
+        managedLocation: tab.managedLocation,
+        dbProperties: tab.dbProperties,
+        isActivated: tab.isActivated,
+        isUnityCatalogSupports,
+        dbVersion
+    });
+
+    if (dbVersion >= UNITY_TAGS_RUNTIME_REQUIRED) {
+        unityCatalogTags = getCatalogTagsStatement(tab);
+        unitySchemaTags = getSchemaTagsStatement(tab, name);
+    }
+
+    return [createStatement, unityCatalogTags, unitySchemaTags].join('\n\n');
```

Comment:
Let's reduce the number of empty lines if tags are not present in the script.
```suggestion
    return [createStatement, unityCatalogTags, unitySchemaTags].filter(Boolean).join('\n\n');
```